sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.net461.cs

```diff
@@ -1,5 +1,20 @@
 namespace Azure
 {
+    public partial class CloudMachine
+    {
+        public CloudMachine(Microsoft.Extensions.Configuration.IConfiguration configuration) { }
+        public CloudMachine(System.IO.Stream configurationContent) { }
+        public CloudMachine(string configurationFile = ".\\cloudconfig.json") { }
+        public string DisplayName { get { throw null; } set { } }
+        public string Id { get { throw null; } }
+        public string Region { get { throw null; } }
+        public string SubscriptionId { get { throw null; } }
```

Comment:
Would it make sense to make this a Guid?

Reply:
It's for integration with https://learn.microsoft.com/en-us/dotnet/api/azure.core.resourceidentifier.subscriptionid?view=azure-dotnet, which is a string. And now I am curious why. @m-nash?